cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,61 +1,117 @@
 "use client";
+import { getBlurredCv } from "@/app/actions/cvs/getBlurCv";
 import Image from "next/image";
 import Link from "next/link";
-import generateImageUrl from "@/helpers/imageURLHelper";
 import Categories from "@/types/models/categories";
-import { useEffect, useState, cache } from "react";
-import { getBlurredCv } from "@/app/actions/cvs/getBlurCv";
+import { useState, useEffect, cache } from "react";
+import { getIdFromLink, getGoogleImageUrl } from "@/helpers/imageURLHelper";
+import {
+  getImageURL,
+  revalidatePreview,
+  getUserName,
+} from "@/app/actions/cvs/preview";
+import Definitions from "@/lib/definitions";
 
-const getBlur = cache(async (imageURL: string) => {
-  return await getBlurredCv(imageURL);
-});
 interface CVCardProps {
   cv: CvModel;
 }
 
+const getBlur = cache(async (imageURL: string) => {
```

Comment:
All of these are obsolete if the function is Client component.
See: https://react.dev/reference/react/cache
TL;DR: use `useMemo`.